analysis/include/PhysObject.hpp

```diff
@@ -81,6 +84,20 @@ class PhysObject {
   int  GetZside()     const;
   double GetSeedTime()      const;
   
+  //For soft muons
+  bool IsGood()     const;
+  int  GetTrkLayers() const;
+  int  GetPixelLayers() const;
+  bool TrkQuality() const;
+  double GetInnerD0() const;
+  double GetInnerDz() const;   
+/////////////////////////////////////////date:5/02/2023, Vertex info
+  int    GetNVertex() const;
+  double GetPVertexX() const;
+  double GetPVertexY() const;
+  double GetPVertexZ() const;
```

Comment:
What does "P" stand for in this variable name? Is it "Primary", or something else? I would vote for not using abbreviations which are not absolutely obvious.